bundle/config/mutator/resourcemutator/apply_bundle_permissions_test.go

```diff
@@ -72,58 +72,82 @@ func TestApplyBundlePermissions(t *testing.T) {
 	require.NoError(t, diags.Error())
 
 	require.Len(t, b.Config.Resources.Jobs["job_1"].Permissions, 3)
-	require.Contains(t, b.Config.Resources.Jobs["job_1"].Permissions, resources.Permission{Level: "CAN_MANAGE", UserName: "TestUser"})
-	require.Contains(t, b.Config.Resources.Jobs["job_1"].Permissions, resources.Permission{Level: "CAN_VIEW", GroupName: "TestGroup"})
-	require.Contains(t, b.Config.Resources.Jobs["job_1"].Permissions, resources.Permission{Level: "CAN_MANAGE_RUN", ServicePrincipalName: "TestServicePrincipal"})
+	require.Contains(
+		t,
+		b.Config.Resources.Jobs["job_1"].Permissions,
+		resources.JobPermission{Level: resources.JobPermissionLevelCanManage, UserName: "TestUser"},
```

Comment:
Can you omit the constants and use immediate strings like below?

If these switch to constants, I would prefer switching all of them to constants.